forum/qa-include/qa-base.php

```diff
@@ -182,11 +182,6 @@ function qa_initialize_constants_1()
 		define('QA_THEME_DIR', QA_BASE_DIR.'qa-theme/');
 		define('QA_PLUGIN_DIR', QA_BASE_DIR.'qa-plugin/');
 
-		if (!file_exists(QA_BASE_DIR.'qa-config.php'))
-			qa_fatal_error('The config file could not be found. Please read the instructions in qa-config-example.php.');
-
-		require_once QA_BASE_DIR.'qa-config.php';
-

```

Comment:
czemu to usunąłeś?

Reply:
bo plik jest dołączony wcześniej, bo wcześniej potrzebna była konfiguracja z niego, więc nie ma sensu dołączać go w tym miejscu